From: Scott Kitterman <scott@kitterman.com>
Date: Tue, 31 Jan 2023 13:41:33 +0000
Subject: Patch out Canonical support information used for Ubuntu - We're
 Debian

Origin: vendor
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=791875
Forwarded: not-needed
Last-Update: 2016-07-14

While in theory this could be managed with dpkg-vendor and some IFDEFery,
it's just easier to patch it out.  This code hasn't changed in years, so the
patch shouldn't take any effort to maintain.
---
 src/DetailsTabs/MainTab.cpp | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/src/DetailsTabs/MainTab.cpp b/src/DetailsTabs/MainTab.cpp
index 1b865f7..db3441c 100644
--- a/src/DetailsTabs/MainTab.cpp
+++ b/src/DetailsTabs/MainTab.cpp
@@ -179,19 +179,19 @@ void MainTab::refresh()
     m_packageShortDescLabel->setText(m_package->shortDescription());
 
     m_descriptionBrowser->setText(m_package->longDescription());
-
+    // Patch out Ubuntu only support information
     // Append a newline to give a bit of separation for the support string
-    m_descriptionBrowser->append(QString());
-    if (m_package->isSupported()) {
-        QDateTime endDate = m_package->supportedUntil();
-        m_descriptionBrowser->append(i18nc("@info Tells how long Canonical, Ltd. will support a package",
-                                        "Canonical provides critical updates for %1 until %2.",
-                                        m_package->name(), endDate.date().toString()));
-    } else {
-        m_descriptionBrowser->append(i18nc("@info Tells how long Canonical, Ltd. will support a package",
-                                        "Canonical does not provide updates for %1. Some updates "
-                                        "may be provided by the Ubuntu community.", m_package->name()));
-    }
+    //m_descriptionBrowser->append(QString());
+    //if (m_package->isSupported()) {
+    //    QDateTime endDate = m_package->supportedUntil();
+    //    m_descriptionBrowser->append(i18nc("@info Tells how long Canonical, Ltd. will support a package",
+    //                                    "Canonical provides critical updates for %1 until %2.",
+    //                                    m_package->name(), endDate.date().toString()));
+    //} else {
+    //    m_descriptionBrowser->append(i18nc("@info Tells how long Canonical, Ltd. will support a package",
+    //                                    "Canonical does not provide updates for %1. Some updates "
+    //                                    "may be provided by the Ubuntu community.", m_package->name()));
+    //}
 }
 
 void MainTab::emitSetInstall()
